compiler/rustc_mir/src/transform/dest_prop.rs

```diff
@@ -126,13 +126,9 @@ const MAX_BLOCKS: usize = 250;
 pub struct DestinationPropagation;
 
 impl<'tcx> MirPass<'tcx> for DestinationPropagation {
-    fn run_pass(&self, tcx: TyCtxt<'tcx>, body: &mut Body<'tcx>) {
-        // Only run at mir-opt-level=2 or higher for now (we don't fix up debuginfo and remove
-        // storage statements at the moment).
```

Comment:
I think it would be good to retain this comment explaining why this optimization is gated on mir-opt-level=2